ajax/libs/jsrsasign/7.1.2/jsrsasign-all-min.js

```diff
@@ -0,0 +1,289 @@
+/*
+ * jsrsasign(all) 7.1.3 (2017-04-16) (c) 2010-2017 Kenji Urushima | kjur.github.com/jsrsasign/license
```

Comment:
How do you get v7.1.3? I thought you simply fetched them via git auto-updater by a special rule and then changed their filename?